WBooks/Screens/MainViewController.swift

```diff
@@ -0,0 +1,60 @@
+//
+//  MainViewController.swift
+//  WBooks
+//
+//  Created by Nacho 2 on 20/01/2020.
+//  Copyright © 2020 Wolox. All rights reserved.
+//
+
+import UIKit
+import WolmoCore
+
+class MainViewController: UIViewController, UITableViewDelegate, UITableViewDataSource {
```

Comment:
Some good practices when extending multiple protocols are:

1) Using "MARK:" as a heading for different sections

```Swift
    class MainViewController: UIViewController, UITableViewDelegate, UITableViewDataSource {    

       // MARK: - UITableViewDelegate

       [Here you add methods related with UITableViewDelegate...]

       // MARK: - UITableViewDataSource
      
       [Here you add methods related with UITableViewDataSource...]
    }
```

2) You can also divide it with extensions:

```Swift
class MainViewController: UIViewController {
    // Code related to UIViewController
}

extension MainViewController: UITableViewDelegate {
    // Here you add methods related with UITableViewDelegate...
}

extension MainViewController: UITableViewDataSource {
    // Here you add methods related with UITableViewDataSource...
}
```

Reply:
Nice! i'll try

Reply:
Actually, the "MARK:" for UIViewController and MainViewController are not necessary, because "MARK:" is mainly used for protocols. Anyway, everything looks great! 🙂